.travis.yml

```diff
@@ -6,11 +6,16 @@ branches:
   only:
     - "master"
 
+services:
+  - docker
+
 install:
-  - pip install -r requirements.txt
+  - docker build -t cd-bot .
+  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then docker run -d -e BOT_TOKEN --name cd-bot cd-bot; fi'
```

Comment:
Have you tested the use of environment variables this way? I'm not very well versed with Travis, but surely `BOT_TOKEN` should be `$BOT_TOKEN` or `"${BOT_TOKEN}"` or something of the sort.

Reply:
This is directly taken from the Travis website : https://docs.travis-ci.com/user/pull-requests/#pull-requests-and-security-restrictions

Reply:
The Travis page you linked to refers to the bash conditional. I'm not referring to the conditional, I'm referring to how you're incorrectly trying to reference the `BOT_TOKEN` environment variable. Environment variables are usually referenced with a dollar sign before the variable name, the same way the Travis example shows you. `BOT_TOKEN` should be changed to `$BOT_TOKEN`.

Here's an example of how to use environment variables this way:
![image](https://user-images.githubusercontent.com/25161069/46456658-8df66080-c7a7-11e8-8098-da565628c91c.png)
The first example is what you're doing currently, and the second is what it should be.

Reply:
Ah, just read up on the documentation for the `docker run` command, looks like I misunderstood  :eyes: